src/python/pants/backend/experimental/python/lockfile_metadata_test.py

```diff
@@ -55,18 +57,34 @@ def line_by_line(b: bytes) -> list[bytes]:
     assert line_by_line(result) == line_by_line(expected)
 
 
-_requirements = ["flake8-pantsbuild>=2.0,<3", "flake8-2020>=1.6.0,<1.7.0"]
-
-
 @pytest.mark.parametrize(
     "requirements,expected",
     [
         ([], "c8e8d0a6d6ec36bee3942091046d81d86e3b83b143b37a7cc714e2d022bf4f85"),
-        (_requirements, "66327c52225d2f798ffad7f092bf1b51da8a66777f3ebf654e2444d7eb1429f4"),
+        (
+            ["flake8-pantsbuild>=2.0,<3", "flake8-2020>=1.6.0,<1.7.0"],
+            "ef417e5ccadca6983fc8ec7df2c8c3e218af029a61752d38bd01926b2adb85b9",
+        ),
     ],
 )
 def test_invalidation_digest(requirements, expected) -> None:
-    assert calculate_invalidation_digest(FrozenOrderedSet(requirements)) == expected
+    a = "flake8-pantsbuild>=2.0,<3"
```

Comment:
Stylistically, you can re-write this test in a parametrized way by using the params:

```
def test_invalidation_digest(reqs_1, reqs_2, are_equivalent):
    assert (calculate_invalidation_digest(reqs_1) == calculate_invalidation_digest(reqs_2)) == are_equivalent
```

That will give you the benefit of the test runner showing all failure cases if things start going wrong, otherwise a failure early on hides the extent of things being broken.

If you're worried about the bit where you generate test cases using a `for` loop, note that the arguments to `parametrize` _could_ be generated by another function.

Reply:
Ack that's possible. Imo, it's simpler to have this current style, e.g. the variables `a`, `b`, and `c` being locally scoped. Keep it simple here.